standalone-e2e-test/src/test/java/GetAllUsersTest.java

```diff
@@ -0,0 +1,75 @@
+import io.restassured.RestAssured;
+import io.restassured.path.json.JsonPath;
+import org.json.JSONException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.skyscreamer.jsonassert.JSONAssert;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+import static org.hamcrest.Matchers.equalTo;
+
+public class GetAllUsersTest extends AbstractRestAssuredBase {
+    @BeforeAll
+    public static void setup() {
+        RestAssured.basePath="api/v1/users";
+    }
+
+
+    @Test
+    void testDefaultGetAllUsers() {
+        RestAssured.given()
+                .log().all()
+                .get()
+                .prettyPeek()
+                .then()
+                .statusCode(200)
+                .body("size()", equalTo(2)) //array size in the root -> number of user records
+                .body("[0].id", equalTo(1))  // spotcheck first ID
+                .body("[0].name", equalTo("John Doe"))  // spotcheck first name
+                .body("[0].links.size()", equalTo(1))  // check number of links returned
+                .body("[1].id", equalTo(2));  // spotcheck second ID
+    }
+
+    @Test
+    void testDefaultGetAllUsersViaExtract() {
+        String responseString = RestAssured.given()
+                .log().all()
+                .get()
+                .prettyPeek()
+                .then()
+                .statusCode(200)
+                .extract()
+                .asString();
+
+        JsonPath js = new JsonPath(responseString);
```

Comment:
You could also try to deserialize the response as an java object.
https://www.toolsqa.com/rest-assured/deserialize-json-response/

Reply:
Added test class testDefaultGetAllUsersWithDeserializedJavaObject to demonstrate this method